crates/miden-tx/src/executor/exec_host.rs

```diff
@@ -159,7 +159,7 @@ where
     fn on_tx_fee_computed(
         &self,
         fee_asset: FungibleAsset,
-    ) -> Result<Vec<AdviceMutation>, TransactionKernelError> {
+    ) -> Result<Vec<AdviceMutation>, TransactionExecutorError> {
```

Comment:
@PhilippGackstatter can probably provide more detail on this (or correct me if I'm wrong), but I don't think we need to change the error type here. Instead, we probably should map the event correctly in the `map_execution_error()` function.